### PR TITLE
Reformat validateaddress help, add missing iswatchonly description.

### PR DESCRIPTION
**Pull request by paveljanik**
Align validateaddress help. Add missing iswatchonly property description.

### CODE REVIEW

File: src/rpcmisc.cpp [hunk 1]
```diff
@@ -157,13 +157,14 @@ UniValue validateaddress(const UniValue& params, bool fHelp)
             "1. \"bitcoinaddress\"     (string, required) The bitcoin address to validate\n"
             "\nResult:\n"
             "{\n"
-            "  \"isvalid\" : true|false,         (boolean) If the address is valid or not. If not, this is the only property returned.\n"
```
Comment (TheBlueMatt) [inline src/rpcmisc.cpp]: Looks like you broke the spacing-alignment here?

Comment (paveljanik) [inline src/rpcmisc.cpp]: Please check spacing-alignment in the `bitcoin-cli help validateaddress`, not in the source code. These lines contain various numbers of escaped `"` characters...

**Comment by TheBlueMatt** [comment]
utACK

**Comment by jonasschnelli** [comment]
ACK

**Pull request merged**